src/erlcloud_ec2.erl

```diff
@@ -1188,6 +1188,7 @@ extract_image(Node) ->
      {description, get_text("description", Node)},
      {root_device_type, get_text("rootDeviceType", Node)},
      {root_device_name, get_text("rootDeviceName", Node)},
+     {creation_date, erlcloud_xml:get_time("creationDate", Node)},
```

Comment:
I think it's from http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Image.html 

> creationDate
>
>    The date and time the image was created.
>   Type: String
>   Required: No